crates/neon-build/src/lib.rs

```diff
@@ -31,6 +31,75 @@ cfg_if! {
                 println!("cargo:rustc-cdylib-link-arg=/DELAYLOAD:node.exe");
             }
         }
+    } else if #[cfg(windows)] {
+        // ^ automatically not neon-sys
+        use std::io::{Error, ErrorKind, Result};
+        use std::process::Command;
+        use std::path::Path;
+
+        fn node_version() -> Result<String> {
+            let output = Command::new("node").arg("-v").output()?;
+            if !output.status.success() {
+                let hopefully_stack_trace = String::from_utf8(output.stderr)
+                    .unwrap_or_else(|_| "<subprocess output garbage>".to_string());
```

Comment:
Could use `String::from_utf8_lossy(output.stderr)` or `std::io::stdout().write_all(output.stderr)` to try a little harder at printing the output.